cmd/main_test.go

```diff
@@ -5,7 +5,7 @@ import "os"
 func Example_version() {
 	oldArgs := os.Args
 	defer func() { os.Args = oldArgs }()
-	os.Args = []string{"kubernetes-mcp-server", "--version"}
+	os.Args = []string{"kubernetes-mcp-server", "version"}
```

Comment:
there's a copy-paste reminiscence from my initial commit here, `kubernetes-mcp-server` should be `ai-cli`(NOTE FOR SELF)